components/organisms/BlogSection/blog-section.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+
+import { DevToArticleResults } from './blog-section.types';
+import { BlogCard } from '@components/atoms';
+
+export function BlogSection() {
+    const [blogData, setBlogData] = useState<DevToArticleResults[] | []>([]);
+
+    useEffect(() => {
+        const getBlogs = async () => {
```

Comment:
You could move this get blogs function outside of the component. Might make things a little cleaner to read.